src/Sarif.Viewer.VisualStudio.UnitTests/ImportanceToForegroundConverterTests .cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Possibly because I filed this PR before @kschecht 's was merged and I did a fetch and rebase to fix merge conflicts?